contracts/src/emergency/sequencer_uptime_feed.cairo

```diff
@@ -0,0 +1,358 @@
+impl Felt252TryIntoBool of TryInto::<felt252, bool> {
+    fn try_into(self: felt252) -> Option<bool> {
+        if self == 0 {
+            return Option::Some(false);
```

Comment:
apparently, only match zero is supported, hence why we use if/else

`error: Only match zero (match ... { 0 => ..., _ => ... }) is currently supported`